exercises/error-handling/.meta/hints.md

```diff
@@ -0,0 +1,11 @@
+This exercise requires you to handle exceptions. An [exception](https://docs.oracle.com/javase/tutorial/essential/exceptions/definition.html) is an event, which occurs during the execution of a program, that disrupts the normal flow of the program's instructions.
+
+In Java, there are two types of exceptions: checked and unchecked exceptions.
+
+Checked exceptions are the exceptions that are checked at compile time.
+
+Unchecked exceptions are the exceptions that are not checked at compiled time.
```

Comment:
Again it might be worth expanding on this a bit to explain what this means (that you don't have to add the exception to method signatures and you don't have to handle it) and explain what kind of errors it's meant to be used for (any that can't be handled at runtime, e.g. running out of memory) 🙂